packages/core/bootstrap/src/lib/burst-limit/reducer.ts

```diff
@@ -3,10 +3,12 @@ import { makeId } from '../rate-limit'
 import * as actions from './actions'
 
 export enum IntervalNames {
+  // SECOND = 'SECOND',
```

Comment:
Can this be removed?

Reply:
I'm about to enable them within the next hour, just want to see a before / after cpu & memory usage of tracking by second.